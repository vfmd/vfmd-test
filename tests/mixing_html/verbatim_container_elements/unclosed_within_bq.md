## pre

> Some text <pre>
> 
> ### Inside pre
> 
> (pre unclosed)

## style

> Some text <style>
> 
> ### Inside style
> 
> (style unclosed)

## script

> Some text <script>
> 
> ### Inside script
> 
> (script unclosed)

End of document
