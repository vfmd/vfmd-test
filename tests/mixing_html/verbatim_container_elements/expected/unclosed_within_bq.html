<h2>pre</h2>
<blockquote>
  Some text <pre>

  <h3>Inside pre</h3>
  <p>(pre unclosed)</p>

</blockquote>
<h2>style</h2>
<blockquote>
  Some text <style>

  <h3>Inside style</h3>
  <p>(style unclosed)</p>

</blockquote>
<h2>script</h2>
<blockquote>
  Some text <script>

  <h3>Inside script</h3>
  <p>(script unclosed)</p>

</blockquote>
<p>End of document</p>

